examples/src/bin/builder_basics.rs

```diff
@@ -0,0 +1,46 @@
+//! # Builder Basics Sample
+//!
+//! This sample demonstrates how to use the builder with an imported UI file
+
+use gio::prelude::*;
+use gtk::prelude::*;
+
+use gtk::{Application, ApplicationWindow, Builder, Button, MessageDialog, ResponseType};
+
+use std::env::args;
+
+fn build_ui(application: &Application) {
+    let glade_src = include_str!("ui/builder_basics.ui");
+    let builder = Builder::from_string(glade_src);
```

Comment:
```suggestion
    let ui_src = include_str!("ui/builder_basics.ui");
    let builder = Builder::from_string(ui_src);
```
We shouldn't mention glade at all, it's not gtk4 compatible